Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -4715,6 +4715,7 @@ void WebPageProxy::receivedNavigationActionPolicyDecision(WebProcessProxy& proce
         RefPtr pageClientProtector = pageClient();
         RefPtr provisionalPage = m_provisionalPage;
         Ref processNavigatingFrom = preferences().siteIsolationEnabled() && frame->isMainFrame() && provisionalPage ? provisionalPage->process() : frame->process();
```

Comment:
I think this would look nicer as:
```
Ref processNavigatingFrom = [&] {
    RefPtr provisionalPage = m_provisionalPage;
    return preferences().siteIsolationEnabled() && frame->isMainFrame() && provisionalPage ? provisionalPage->process() : frame->process();
}();
```